### DISCUSSION TITLE
Installing expo package in starter monorepo

### DISCUSSION THREAD

**Discussion by tamagutchi** [tamagui/tamagui#1144] [Q&A]
I am trying to add [expo-av](https://docs.expo.dev/versions/latest/sdk/av/) package. I installed it in `packages/app`, and then added it to `transpilePackages` in `next.config.js` however I am not able to use it on web (it's working in native app). The error I am getting is:

```
Module parse failed: Unexpected token (13:7)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 'use strict';
| 
> export type PackagerAsset = {
|   +__packager_asset: boolean,
|   +fileSystemLocation: string,

Import trace for requested module:
../../node_modules/@react-native/assets/registry.js
../../node_modules/react-native/Libraries/Image/AssetRegistry.js
../../node_modules/expo-asset/build/Asset.js
../../node_modules/expo-asset/build/index.js
../../node_modules/expo-av/build/AV.js
../../node_modules/expo-av/build/Audio.js
../../node_modules/expo-av/build/index.js
../../packages/ui/src/video.tsx
../../packages/ui/src/index.tsx
../../packages/app/provider/index.tsx
```

Any hints?

**Comment by matthewmturner** [comment]
FYI see this conversation. I haven't tried the latest recommendation yet but I'm hopeful. 

https://github.com/tamagui/tamagui/issues/846#issuecomment-1541553343

**Reply by tamagutchi** [reply]
Thank you for the answer, unfortunately it does not seem to work.

**Comment by TomasSestak** [comment]
same issue, did you find a solution?

**Comment by rdewolff** [comment]
I am facing the same issue. Did you find a solution?
